basic/components/hello_command_component/spec.yaml

```diff
@@ -0,0 +1,14 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
+type: command
+name: hello_command_component
+version: 0.0.1
+display_name: Hello Command Component
+is_deterministic: false
```

Comment:
>false [](http://example.com/codeflow?start=18&length=5)

Is there a reason to mark these as non-deterministic?

Reply:
Not necessarily in this case, will make them deterministic!